### ISSUE TITLE
Field value ignored when passed as ActiveCampaign tag

### ISSUE THREAD

**Issue by jimirobaer** [verbb/formie#654]
**Description**

Running into a strange issue where any value passed to ActiveCampaign's Tag field (after setting up the integration) seems to result in two tags being added: "Another" and "Mailchimp".  I pass the default value from a hidden field as the field used for 'Tags' in Formie. 

I have reproduced this behaviour on two seperate Craft CMS + Formie installations, and two separate ActiveCampaign accounts.

After submitting, the value of the hidden field is correct when reading the submission in Craft CMS, so I guess something is going wrong after running the ActiveCampaign integration.

If there's a way to see the detailed request that's being sent to ActiveCampaign I could debug further myself, but I can only confirm the integration succeeds after submitting.

**Steps to reproduce**
1. Setup 'ActiveCampaign' integration
2. Create a form with a hidden field 'Tags' and activate the 'ActiveCampaign' integration.
3. Link the hidden field to ActiveCampaigns 'Tags' field after refreshing the list.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax & Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.26
- Craft version: 3.7.21
- Multi-site: No

**Additional context**

![Screenshot 2021-11-23 at 13-40-19 SKINN31230 Email Marketing - Contact View](https://user-images.githubusercontent.com/8984769/143025734-ab63564c-8402-4f78-bcb1-4588ec0fb327.png)

![Screenshot 2021-11-23 at 13-41-04 🚧⚙ 2021-11-23 12 46 32 - SKINN](https://user-images.githubusercontent.com/8984769/143025763-b1fc13cf-decc-4a9f-a480-1363d9ff5484.png)

<img width="959" alt="Screenshot 2021-11-23 at 13 46 39" src="https://user-images.githubusercontent.com/8984769/143026434-00d0db57-f28e-45ee-8030-228d51257f5e.png">

**Comment by engram-design** [comment]
Ah - so sorry about that. Looks like a simple (stupid) case of leaving my debugging info in the integration that added these two fields. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.26",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.27](https://github.com/verbb/formie/releases/tag/1.4.27)
